rioxarray/merge.py

```diff
@@ -132,7 +132,7 @@ def merge_arrays(
     precision: float, optional
         Number of decimal points of precision when computing inverse transform.
     method: str or callable, optional
-        See rasterio docs.
+        See rasterio docs (https://rasterio.readthedocs.io/en/stable/api/rasterio.merge.html).
```

Comment:
I'm not familiar with what rioxarray uses for sphinx docs, but theoretically you could put

```
:mod:`rasterio.merge`
```

and when rendered to the sphinx HTML is should automatically map to the API docs in rasterio. This won't show up in the raw string docstring, but maybe something @snowman2 is interested in.

Reply:
As @djhoese mentioned, you could follow the pattern shown here: https://github.com/corteva/rioxarray/blob/94d3d53d0e3bfd12a7c3d1ce7fffd91fc7f5e141/rioxarray/raster_dataset.py#L89

And do something like this:
```
See :func:`rasterio.merge.merge` for more details.
```

Reply:
@chbrandt, what are your thoughts on modifying this PR with the suggestions above?

Reply:
Agree, @snowman2